runtime/doc/syntax.txt

```diff
@@ -946,7 +946,7 @@ Enables Stylus for ".astro" files. Default Value: "disable"
 NOTE: You need to install an external plugin to support stylus in astro files.
 
 
-ASPPERL and ASPVBS			*ft-aspperl-syntax* *ft-aspvbs-syntax*
+ASPPERL AND ASPVBS			*ft-aspperl-syntax* *ft-aspvbs-syntax*
 
```

Comment:
These two might look better on separate lines, as is done for DOCBOOK, i.e.,
`-ASPPERL and ASPVBS			*ft-aspperl-syntax* *ft-aspvbs-syntax*`
`+ASPPERL				*ft-aspperl-syntax*`
`+ASPVBS				*ft-aspvbs-syntax*`

Reply:
Separated.